package-lock.json

```diff
@@ -1,30 +1,51 @@
 {
   "name": "tdr-auth-server",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Are we doing this upgrade to `lockfileVersion` 2?

Reply:
I didn't  mean to, it must have done it automatically. I've changed it back to 1 and it doesn't seem to have caused any problems so I'll leave it as 1.